src/libraries/Common/src/System/Security/Cryptography/X509IterationCountExceededException.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Security.Cryptography.X509Certificates
+{
+    internal sealed class X509IterationCountExceededException : Exception
```

Comment:
should this derive from CryptographicException?

Reply:
No, this type is just a placeholder for the "PKCS12 (PFX) without a supplied password has exceeded maximum allowed iterations" error.